gravityflow/gravityflow
# Add HubSpot step for the Gravity Forms HubSpot add-on

This PR adds a new HubSpot step for the Gravity Forms HubSpot add-on.

## Testing instructions
1. Create a form with a HubSpot feed.
2. Add a new HubSpot step to the workflow, and add an Approval step before it.
3. After submitting the form, make sure the email only get subscribed after approval.
